mir-module/src/main/resources/xsl/facets.xsl

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:encoder="xalan://java.net.URLEncoder"
+                xmlns:i18n="xalan://org.mycore.services.i18n.MCRTranslation"
+                xmlns:mcrxsl="xalan://org.mycore.common.xml.MCRXMLFunctions"
+                xmlns:xalan="http://xml.apache.org/xalan"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                exclude-result-prefixes="i18n mcrxsl encoder fn"
+>
+  <xsl:template name="facets">
+    <xsl:if test="/response/lst[@name='facet_counts']/lst[@name='facet_fields'] !=''">
+      <xsl:for-each select="/response/lst[@name='facet_counts']/lst[@name='facet_fields']/*">
+        <xsl:variable name="facet_name" select="self::node()/@name"/>
+        <!-- TODO: remove conditions for facets 'worldReadableComplete' and 'mods.genre' after code refactoring -->
+        <xsl:if test="self::node()[@name=$facet_name]/int">
+          <div class="card {$facet_name}">
+            <div class="card-header" data-toggle="collapse-next">
+              <h3 class="card-title">
+                <!-- Checking facet name for compatibility with old code, facets named -->
+                <!-- 'worldReadableComplete' and 'mods.genre' from old code -->
+                <xsl:choose>
+                  <!-- facet 'worldReadableComplete' -->
+                  <xsl:when test="$facet_name='worldReadableComplete'">
+                    <xsl:value-of select="i18n:translate('mir.response.openAccess.facet.title')"/>
+                  </xsl:when>
+
+                  <!-- facet 'mods.genre' -->
+                  <xsl:when test="$facet_name='mods.genre'">
+                    <xsl:value-of select="i18n:translate('editor.search.mir.genre')"/>
+                  </xsl:when>
+
+                  <!-- all other facets -->
+                  <xsl:otherwise>
```

Comment:
For classification one could use the name of the classification as default label, when a specific i18n key is not available

Reply:
ok, this is exactly what I'm doing now

Reply:
> ok, this is exactly what I'm doing now

I would prefer the label of the classification in $CurrentLang as title for the facet. The same applies for the facet values. The ids of the categories should be translated to their labels in $CurrentLang.